drivers/SmartThings/matter-sensor/profiles/air-quality-sensor-common.yml

```diff
@@ -0,0 +1,20 @@
+name: air-quality-sensor-common
+components:
+  - id: main
+    capabilities:
+      - id: airQualityHealthConcern
+        version: 1
+      - id: temperatureMeasurement
+        version: 1
+      - id: relativeHumidityMeasurement
+        version: 1
+      - id: carbonDioxideMeasurement
+        version: 1
+      - id: dustSensor
+        version: 1
+      - id: tvocMeasurement
+        version: 1
+      - id: refresh
```

Comment:
Every new profile should include the `firmwareUpdate` capability listed before `refresh`

Reply:
Done in [c95fe0e](https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/pull/1422/commits/c95fe0edd88512aa38632d4a9a73eded6dd5314d)!